posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -170,14 +177,18 @@ async def iter_model_records(
     client: ClickHouseClient,
     model: BatchExportModel | BatchExportSchema | None,
     team_id: int,
-    is_backfill: bool,
     interval_start: str | None,
     interval_end: str,
     destination_default_fields: list[BatchExportField] | None = None,
     # TODO - remove this once all batch exports are using the latest schema
     use_latest_schema: bool = False,
+    # TODO: this can be removed once all backfill inputs are migrated
+    is_backfill: bool = False,
+    backfill_details: BackfillDetails | None = None,
```

Comment:
Similar nitpick as before, to make it clear what is to delete, move `backfill_details` up: 
```suggestion
    backfill_details: BackfillDetails | None = None,
    # TODO - remove this once all batch exports are using the latest schema
    use_latest_schema: bool = False,
    # TODO: this can be removed once all backfill inputs are migrated
    is_backfill: bool = False,
```